ibis/backends/dask/execution/util.py

```diff
@@ -37,3 +39,55 @@ def make_selected_obj(gs: SeriesGroupBy):
         return gs.obj.set_index(gs.index, drop=False)[
             gs._meta._selected_obj.name
         ]
+
+
+def maybe_wrap_scalar(result: Any, expr: ir.Expr) -> Any:
+    """
+    A partial implementation of `coerce_to_output` in the pandas backend.
+
+    Currently only wraps scalars, but will change when udfs are added to the
+    dask backend.
+    """
+    result_name = expr.get_name()
+    if isinstance(result, dd.core.Scalar) and isinstance(
+        expr.op(), ops.Reduction
+    ):
+        # TODO - computation
+        return dd.from_pandas(
+            pd.Series(result.compute(), name=result_name), npartitions=1
+        )
+        # return dd.Series(result, name=result_name)
+    else:
+        return result.rename(result_name)
+
+
+def safe_concat(dfs: List[Union[dd.Series, dd.DataFrame]]) -> dd.DataFrame:
+    """
+    Concat a list of `dd.Series` or `dd.DataFrame` objects into one DataFrame
```

Comment:
Are the Series and DataFrame aligned w.r.t partitiong?

Reply:
Not entirely sure what you mean - this function itself does not enforce any sort of partitioning.